bounded/src/sat.rs

```diff
@@ -3,6 +3,7 @@
 
 //! A bounded model checker for flyvy programs using a SAT solver.
```

Comment:
Can you mention cadical here now? One thing I want to clarify here is that Z3 isn't being used.

```suggestion
A bounded model checker for flyvy programs using the [CaDiCaL][cadical] SAT solver.

[cadical]: https://fmv.jku.at/cadical/
```